mock/py/mockbuild/mounts.py

```diff
@@ -159,6 +159,9 @@ def __init__(self, rootObj):
             device = 'mock_hide_{}fs_from_host'.format(mount)
             host_path = rootObj.make_chroot_path(mount_point)
 
+            if rootObj.bootstrap_buildroot:
+                host_path = rootObj.bootstrap_buildroot.make_chroot_path(host_path)
```

Comment:
Ok, this isn't a good approach.  In the actual situation, we need to have the /proc
mounted both in `/var/lib/mock/<chroot>/root/proc` (because rpmbuild is executed
there) and also in `/var/lib/mock/<chroot>-bootstrap/root/var/lib/mock/<chroot>/root/proc`
because the bootstrapped DNF operates there.